components/list/list-item-nav-button-mixin.js

```diff
@@ -72,6 +71,9 @@ export const ListItemNavButtonMixin = superclass => class extends ListItemButton
 		if (this.current) {
 			this.dispatchSetChildCurrentEvent(true);
 		}
+
+		this.addEventListener('focusin', this.#handleFocusIn);
+		this.addEventListener('focusout', this.#handleFocusOut);
```

Comment:
These events are experimental... they should be safe to use but curious whether they're necessary vs. `focus` and `blur`? I'm guessing because these bubble?

Reply:
Oh interesting - we use them in list-item-mixin to set `_focusing` so I was following that pattern. I had also tried `focus` and `blur` earlier and there was some gotcha, but my approach has changed since so I'll retry `focus` and `blur`.

Reply:
Cool -- I don't think it's a big deal if you do end up needing them because of how they bubble, just wanted to make sure it was a conscious choice as I rarely see them used.

Reply:
Tested it out and confirmed that we'd want to use these events for the bubbling.